examples/grpo_climb_engaging.yaml

```diff
@@ -1,6 +1,8 @@
 data:
-  train_files: /scratch/high_modality/geom_train.jsonl
-  val_files: /scratch/high_modality/geom_valid_mini.jsonl
+  # train_files: /scratch/high_modality/geom_train.jsonl
+  # val_files: /scratch/high_modality/geom_valid_mini.jsonl
+  train_files: /mnt/8T/high_modality/geom_train.jsonl
+  val_files: /mnt/8T/high_modality/geom_valid_mini.jsonl
```

Comment:
Since I'm using this file maybe you can create a new .yaml for testing on workstation? Or just revert this.